roles/check_drained_node/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Isn't this the same as `check_drained_node.yml`?

Reply:
Was indeed a duplica (when splitting a big role into smaller one)